hooks/playbooks/tempest_store_timing_data.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Store stestr timing data in repository
+  hosts: localhost
+  gather_facts: false
+  vars:
```

Comment:
nit: I know that in all places we have done same way, because we relay on ansible var order, but normally and good habit would be to remove "vars" and move it to vars defaults, e.g. "{{ timing_data_repo | default('https://repo.example.org/project/timing-data') }}"